### PR TITLE
Rename to `@ember/ordered-set`

### PR DESCRIPTION
**Pull request by Turbo87**
as discussed in https://github.com/emberjs/ember-ordered-set/pull/3#discussion_r149094350

### CODE REVIEW

**Review by rwjblue — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ember-ordered-set
+# @ember/ordered-set
```
Comment (locks) [inline README.md:1]: do we maybe want to leave this one matching with the repo name?

Comment (Turbo87) [reply README.md]: I

Comment (Turbo87) [reply README.md:1]: I'm fine either way, but `ember init` will always change it back to `@ember/ordered-set` 🤷‍♂️

Comment (locks) [reply README.md:1]: :shipit:

**Review by locks — APPROVED** [review]

**Pull request merged**